sdk/Lusid.Sdk/Utilities/LusidApiFactory.cs

```diff
@@ -68,6 +68,15 @@ public LusidApiFactory(Configuration configuration)
 
         private static Dictionary<Type, IApiAccessor> Init(Configuration configuration)
         {
+            // If some retry policy has already been assigned, use it.
+            // Users can combine their own policy with the default policy by using the .Wrap() method.
+            RetryConfiguration.RetryPolicy = 
+                RetryConfiguration.RetryPolicy ?? ApiRetryHandler.GetInternalExceptionRetryPolicy();
+            
+            // If some async retry policy has already been assigned, use it.
+            RetryConfiguration.AsyncRetryPolicy =
+                RetryConfiguration.AsyncRetryPolicy ?? ApiRetryHandler.GetInternalExceptionAsyncRetryPolicy();
```

Comment:
If I instantiate an ApiClientFactory this gets set for every ApiClient both already instantiated and those that will be created in the future, is this correct?

It is it possible to only apply the `ApiRetryHandler` to the instances of the ApiClient produce by this ApiClientFactory?

Reply:
I was originally looking where we could set a default retry policy on startup. We can't do it in the RetryConfiguration.RetryPolicy, because that's part of the generation template.

I was hoping that by creating any API factory, the retry policy would be assigned.

Would we want it to work differently? And why if yes?

Reply:
I can certainly see a use case where you have different ApiClientFactory's with different retry settings. The behaviour might surprise you if you try and change one and they all change.

Reply:
could you give an example of how to remedy this?

Reply:
`RetryPolicy` config is a global static method used in the generated ApiClient class https://github.com/finbourne/lusid-sdk-csharp-preview/blob/master/sdk/Lusid.Sdk/Client/ApiClient.cs#L492

How would you remedy this such the polly retry policy configuration is different for every LusidApiFactory instance, and where would you set the configuration from?

Reply:
Fair enough, this behaviour is consistent with the default behaviour of a global retry policy.